From 101507f74eafaef8f6cf905e1c13a634f205f43c Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Fri, 19 Jan 2024 21:31:33 +0100
Subject: [PATCH] libcheese: Add GtkWidget cast to avoid an
 incompatible-pointer-types error

This is required for compilation with GCC 14.
---
 libcheese/cheese-flash.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libcheese/cheese-flash.c b/libcheese/cheese-flash.c
index 59959dbd3..e40232216 100644
--- a/libcheese/cheese-flash.c
+++ b/libcheese/cheese-flash.c
@@ -132,7 +132,7 @@ cheese_flash_set_property (GObject      *object,
       GObject *parent;
       parent = g_value_get_object (value);
       if (object != NULL)
-        priv->parent = g_object_ref (parent);
+        priv->parent = GTK_WIDGET (g_object_ref (parent));
       else
         priv->parent = NULL;
     }
